fitbenchmarking/fitbenchmarking
# Adding option toggle for 3rd party output

#### Description of Work

Fixes #513 

As noted in #500 the output grabber does not work with Windows currently. In making the output grabber to pass through this case we have added it to the options file to be able to toggle on and off.


#### Testing Instructions

1. Check things run on windows
2. Change `external_output` in the options files to check the expected behaviour is correct 

Function: Does the change do what it's supposed to?

Tests: Does it pass? Is there adequate coverage for new code?

Style: Is the coding style consistent? Is anything overly confusing?

Documentation: Is there a suitable change to documentation for this change?
